Src/VimWpf/VimWpf.csproj

```diff
@@ -10,6 +10,8 @@
     <Reference Include="Microsoft.VisualStudio.CoreUtility, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
     <Reference Include="Microsoft.VisualStudio.Language.Intellisense, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
     <Reference Include="Microsoft.VisualStudio.Language.StandardClassification, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.VisualStudio.Shell.14.0, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />

```

Comment:
The Shell reference shouldn't be necessary here.

Reply:
Right you are! The addition of the `VsVimJoinableTaskFactoryProvider` to the VimWpf project (I put it there by analogy with `VsVimHost` caused some reference issues, starting with compilation errors related to `ThreadHelper` which doesn't have a `JoinableTaskFactory` property in older versions of the Shell assembly. I also ran into one slightly odd error about the options dialog having to derive from `Dialog`, which it clearly did derive from.

I've removed it now.